src/integrity.test.ts

```diff
@@ -1,9 +1,9 @@
-import { stat } from 'fs/promises';
```

Comment:
I'd say the previous import style is preferable, because it (normally) allows for treeshaking

```
import { namedImport } from 'fs/promises';
```

Reply:
It was failing the build for some reason. Will change it back and dig into why.

Reply:
I also started using `fs/promises` quite recently so it could be that the linter complains because I haven't updated the rules yet.

Reply:
![image](https://user-images.githubusercontent.com/1751064/164545100-0022744e-e4b4-41b0-b985-9e2c8b4dccc6.png)

Reply:
@webbertakken any idea what the issue is?

Reply:
Yup, you need to `nvm use 16.13.0` or similar to use a newer node version.

Some of our repos still require 12, but we should upgrade to latest stable.